### PR TITLE
Add MVP emoji page

### PR DESCRIPTION
**Pull request by plaindocs**
image

~I find myself in the strange position of needing to escape the colons in the code blocks, because the order the substitutions happen, probably we should figure out where replace text with emoji happens, and tell it not to do it inside code blocks?~ 

- [x] Add page to left menu
- [ ] We've talked about a search box for emoji using the API (or the json data in the subrepo) but I'd be tempted to merge this and add later.
- [ ] Emoji vote/comment this post to suggest example emoji to be used in the short list :-p

### CODE REVIEW

**Comment by plaindocs** [comment]
I totes forgot about this. @harrietgrace anything to add / remove?

**Comment by harrietgrace** [comment]
Looks super! Can we add a note about how you can PR your own emojis into the repo too?

**Comment by plaindocs** [comment]
Done. Comment soon if you have anything to add ;-)

**Comment by toolmantim** [comment]
How did I never see this until now?? 😍😍

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -204,6 +204,8 @@
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Managing Log Output", 'pipelines/managing-log-output' %></li>
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Links & Images in Log Output", 'pipelines/links-and-images-in-log-output' %></li>
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Notifications", 'pipelines/notifications' %></li>
+                  <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Emoji", 'pipelines/emoji' %></li>
+
```
Comment (toolmantim) [inline app/views/layouts/application.html.erb:208]:
```suggestion
```

**Comment by plaindocs** [comment]
I don't have strong feelings on emoji / emojis at all.
You're entirely correct on the table front, that is just my allergy to writing HTML in inside markdown coming out...

**Pull request merged**